### PR TITLE
Rush log files

### PR DESCRIPTION
**Pull request by mbwhite**
Changes in this PR

- update the pipeline file to include more comments, and the future cron script
- uploads log files, unit, fv and coverage outputs
- uploads the jsdoc API html
- uploads the tgz and docker image
- pulls the latest docker images for the master branch

-

### CODE REVIEW

File: test/fv/crud.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ const expect = chai.expect;
 const utils = require('./utils');
 const {SHORT_STEP, MED_STEP, LONG_STEP} = utils.TIMEOUTS;
 
-describe('Chaincode CRUD', () => {
+describe.only('Chaincode CRUD', () => {
```
Comment (awjh-ibm) [inline test/fv/crud.js]: I guess this isn't deliberate?

File: test/fv/utils.js [hunk 2]
```diff
@@ -33,16 +33,18 @@ async function install(ccName) {
...
         await exec(util.format(cmd, 'org2_cli'));
+
     } finally {
         fs.unlinkSync(npmrc);
     }
 }
 
 async function instantiate(ccName, func, args) {
     const cmd = `docker exec org1_cli peer chaincode instantiate ${getTLSArgs()} -o orderer.example.com:7050 -l node -C mychannel -n ${ccName} -v v0 -c '${printArgs(func, args)}' -P 'OR ("Org1MSP.member")'`;
+    console.log(cmd)
```
Comment (awjh-ibm) [inline test/fv/utils.js]: Deliberate?

File: test/fv/crud.js (hunk 1 above)
Comment (mbwhite) [reply test/fv/crud.js]: it was... to make the build faster to run.. but need to remove.

File: test/fv/utils.js (hunk 2 above)
Comment (mbwhite) [reply test/fv/utils.js]: no

File: libraries/fabric-shim/package.json [hunk 3]
```diff
@@ -37,6 +37,25 @@
   },
   "types": "./types/index.d.ts",
   "license": "Apache-2.0",
+  "nyc": {
+    "exclude": [
```
Comment (heatherlp) [inline libraries/fabric-shim/package.json]: don't think this stanza is quite right, there's no gulpfile, do you want to exclude cli.js? and index.js too?

File: test/fv/package.json [hunk 4]
```diff
@@ -40,6 +40,8 @@
     "mockery": "^2.1.0",
     "rewire": "^4.0.1",
     "sinon": "^6.1.5",
-    "sinon-test": "^2.2.0"
+    "sinon-test": "^2.2.0",
+
```
Comment (heatherlp) [inline test/fv/package.json]: don't think we should have empty lines in a .json

File: tools/azure-mocha-reporter/package.json [hunk 5]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "azure-mocha-reporter",
+  "version": "1.0.0",
+  "description": "Very simple spliter for mocah to Spec (humans) and JUnit format (AzurePipelines)",
```
Comment (heatherlp) [inline tools/azure-mocha-reporter/package.json]: "mocha"

File: tools/toolchain/network/crypto-material/core.yaml [hunk 6]
```diff
@@ -216,18 +220,16 @@ peer:
             responseTimeout: 3s
             # batchSize the number of blocks to request via state transfer from another peer
             batchSize: 10
-            # blockBufferSize reflect the maximum distance between lowest and
-            # highest block sequence number state buffer to avoid holes.
-            # In order to ensure absence of the holes actual buffer size
-            # is twice of this distance
+            # blockBufferSize reflects the size of the re-ordering buffer
+            # which captures blocks and takes care to deliver them in order
+            # down to the ledger layer. The actually buffer size is bounded between
```
Comment (heatherlp) [inline tools/toolchain/network/crypto-material/core.yaml]: actual instead of actually

**Comment by heatherlp** [comment]
This (or a subset/equivalent) of these changes needs porting into the release-1.4 branch

**Pull request merged**